widgets/versionlist/partials/_sort.htm

```diff
@@ -0,0 +1,8 @@
+<div class="pull-right">
+    <?php if($this->getConfig('sort', 'asc') === 'asc'): ?>
+    <a href="javascript:;" data-request="<?= $this->getEventHandler('onSort') ?>" data-request-data="sort: 'desc'" class="white oc-icon-sort-numeric-desc" title="<?= e(trans('rainlab.builder::lang.version.sort_descending')) ?>"></a>
```

Comment:
@filipac Please indent this 4 spaces to the right to nest it inside the condition. Please also do the same for the line below the `<?php else: ?>` line.